next/app/components/common/GenerateButton/GenerateButtion.tsx

```diff
@@ -1,5 +1,16 @@
+'use client'
+
 import React from 'react'
+import s from './GenerateButton.module.css'
 
 export default function GenerateButton() {
-  return <></>
+  return (
+    <section className={s.container}>
+      <div className={s.component}>
+        <div className={s.generateButton}>
+          <p>Generate</p>
+        </div>
+      </div>
+    </section>
```

Comment:
再利用しやすいコンポーネントを作ることが目的のため、sectionタグやdivタグは不要です！
```suggestion
    <button className={s.generateButton}>
      <p>Generate</p>
    </button>
```